README.md

```diff
@@ -274,3 +274,16 @@ To generate API documentation, install doxygen 1.8.7 or later.
     $ doxygen
 
 To view the documentation, open `lib/html/index.html` in a web browser.
+
+Debugging
```

Comment:
Could also be worth mentioning running make in verbose mode to see what tests are failing like:

```
$ make test ARGS=-V
```